test/blog.test.js

```diff
@@ -0,0 +1,83 @@
+var request = require("request"); // setting up the environment - requesting to the server
+var user = require("../app.js");
+var base_url = "http://localhost:3000/blog";
+
+xdescribe("Server",() => {
+  var server;
+  beforeALL(() => {
+    server = require("../app.js");
+    });
+    afterALL(() => {
+      server.close();
+  });
```

Comment:
Recommend moving the code block in ln6..ln12 into the main "myapp" describe block. Having this in a separate describe block limits its scope to just that block - meaning it would only run for "Server" describe block and wont run for myapp.